aero-mavlink-router-samples/debian/copyright

```diff
@@ -0,0 +1,13 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: aero-mavlink-router-samples
+Source: https://github.com/01org/mavlink-router/tree/master/examples
+
+Files: *
+Copyright: 2017 Intel Corporation
+License: GPL-2
```

Comment:
MAVLink Router is Apache-2 - https://github.com/intel/mavlink-router/blob/master/LICENSE

Reply:
Updated license